aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -115,10 +115,10 @@ RegistrationHandleRAII Dispatcher::registerDef(FunctionSchema schema) {
 
   if (op.operatorIterator_->def_count == 0) {
     // NB: registerSchema is not idempotent! Only do it once!
-    op.operatorIterator_->op.registerSchema(std::move(schema));
+    op.operatorIterator_->op.registerSchema(std::move(schema), std::move(debug));
     listeners_->callOnOperatorRegistered(op);
   } else {
-    checkSchemaCompatibility(op, schema);
+    checkSchemaCompatibility(op, schema, debug);
```

Comment:
nit: I'd probably also add the std::move here. It doesn't change anything because the `checkSchemaCompatibility` function takes a reference, but that's not immediately visible at the call site, so the call site is easier to check for consistent `std::move` usage if we just add it everywhere where it is valid. That also future-proofs in case `checkSchemaCompatibility` decides to take it by value later.

Reply:
This doesn't make sense to me. Semantically, it doesn't make any sense for schema compatibility to take ownership of the schema. It is after all just going to read the schema, not store it anywhere. Isn't adding a `std::move` here "just because" it's the terminal use obscuring the semantic meaning of the code here?